kuma/settings/common.py

```diff
@@ -1507,13 +1507,14 @@ def parse_iframe_url(url):
         1000,
         "Number of expired sessions to cleanup up in one go.",
     ),
-    WELCOME_EMAIL_FROM=(
-        "Janet Swisher <no-reply@mozilla.org>",
-        "Email address from which welcome emails will be sent",
-    ),
     AKISMET_KEY=("", "API key for Akismet spam checks, leave empty to disable"),
 )
 
+# Email address from which welcome emails will be sent
+WELCOME_EMAIL_FROM = config(
+    "WELCOME_EMAIL_FROM", default="Janet Swisher <no-reply@mozilla.org>"
```

Comment:
Do we still want to force Janet to be the human-readable face of the automated email whenever anyone signs up?

Reply:
Actually, we should defer that decision, since the welcome emails are written from Janet's first person perspective.

...though they also mention IRC, which doesn't exist anymore. I'll file a separate bug.

Reply:
Good point. This is the current setting on stage and prod, but maybe we should change it to something like `MDN Web Docs Team` since the community aspect of MDN is no longer Janet's role? @jmswisher Do you have a recommendation here?

Reply:
@callahad Oops, missed your previous comment while creating mine!

Reply:
https://github.com/mdn/kuma/issues/6365#issuecomment-577371250

*This* PR doesn't actually take a stand. It just rearranges the Python code (which I know you know)
But to consider;

* Should the welcome email come from "MDN" or "Mozilla Developer Network" or "Mozilla MDN" instead of "Janet Swisher"??
* Having a real persons full name hardcoded in the Python code feels a bit "unprofessional"
* Can anybody remember WHY we picked a human name instead of a more "corporate name"? CC @atopal 

PS. I'm going to go ahead with this PR but we can still discuss the topic on a tagent.

Reply:
I wanted the email to appear to come from a human, to make it more personal. In fact, I wanted it to come from a reply-able email alias like mdn-community-mgr@mozilla.org, instead of an impersonal no-reply address, but that was deemed infeasible. 

I'm fine with continuing to have my name appear on the email, though it does sound like the message needs to be revisited, if it mentions IRC. Something like "Janet Swisher, MDN Community Lead" would be better than just "Janet Swisher", which leads to users wondering WTH "Janet Swisher" is.

Reply:
I remember starting a branch to tidy up that welcome email the day they shut down the IRC but then I got sidetracked because I didn't have a Matrix URL to replace it with. 

I for one think that "Janet Swisher, MDN Community Lead" is a huge improvement with or without fixing the content of the email itself. 
And, let's leave to entirely to you Janet to decide what email you want. I would strongly recommend a new change request issue explicitly for the *value* of the `WELCOME_EMAIL_FROM` config.

Reply:
Separately, I think @atopal and @malqinneh should also scrutinize that email to consider a link to the Subscription landing page. Every $ counts and with some thinking time on the right wording, it's really easy to put in an edit to that email body. And we can actually use waffle flags for it too I think.

Reply:
Separate issue filed at #6883